core/domain/exp_jobs_one_off.py

```diff
@@ -34,13 +34,23 @@
 from core.platform import models
 import feconf
 import python_utils
+import schema_utils
 import utils
 
+from pylatexenc import latex2text
+
 (exp_models,) = models.Registry.import_models([
     models.NAMES.exploration])
 gae_image_services = models.Registry.import_gae_image_services()
 
-SEPARATORS = ['<', '>', '=']
+# This threshold puts a cap on the number of valid explorations that can be
+# yielded by the MathExpressionValidationOneOffJob.
+VALID_MATH_EXP_YIELD_LIMIT = 200
```

Comment:
Do not abbreviate -- there is ambiguity about exploration vs expression

Reply:
Done.